content/en/sensitive_data_scanner/_index.md

```diff
@@ -0,0 +1,178 @@
+
+For Terraform, see the [Datadog Sensitive Data Scanner rule][6] resource.
+
+To add scanning rules:
+
+1. Navigate to [Sensitive Data Scanner][5].
+1. Click the scanning group where you want to add the scanning rules.
+1. Click **Add Scanning Rule**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Rule**.
+1. Select whether you want to add a library rule or create a custom scanning rule.
+
+{{< tabs >}}
+{{% tab "Add scanning rule from the library rules" %}}
+
+The Scanning Rule Library contains predefined rules for detecting common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more.
+
+1. In the **Add library rules to the scanning group** section, select the library rules you want to use.
+1. In the **Define rule target and action** section, select if you want to scan the **Entire Event** or **Specific Attributes**.
+    - If you are scanning the entire event, you can optionally exclude specific attributes from getting scanned.
+    - If you are scanning specific attributes, specify which attributes you want to scan.
+{{% sds-scanning-rule %}}
+1. Click **Add Rules**.
+
+{{% /tab %}}
+{{% tab "Add a custom scanning rule" %}}
+
+You can create custom scanning rules using regex patterns to scan for sensitive data.
+
+1. In the **Define match conditions** section, specify the regex pattern to use for matching against events in the **Define regex** field. Enter sample data in the **Regex tester** field to verify that your regex pattern is valid.   
+    Sensitive Data Scanner supports Perl Compatible RegEx (PCRE), but the following patterns are not supported:
+    - Backreferences and capturing sub-expressions (lookarounds)
+    - Arbitrary zero-width assertions
+    - Subroutine references and recursive patterns
+    - Conditional patterns
+    - Backtracking control verbs
+    - The \C "single-byte" directive (which breaks UTF-8 sequences)
+    - The \R newline match
+    - The \K start of match reset directive
+    - Callouts and embedded code
+    - Atomic grouping and possessive quantifiers
+{{% sds-scanning-rule %}}
+1. Click **Add Rule**.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+**Notes**:
+
+- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
+- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
+- To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.
+
+See [Investigate Sensitive Data Issues][7] for details on how to use the [Summary][8] page to triage your sensitive data issues.
```

Comment:
This link will work after [this PR](https://github.com/DataDog/documentation/pull/21567) is merged into the `may/main-sds-refresh` branch.